Sources/IO/Geometry/PLYReader/index.js

```diff
@@ -157,6 +163,25 @@ function postProcess(buffer, elements) {
     }
   }
 
+  if (!hasVertTCoords && hasFaceTCoords) {
+    // don't use array.shift, because buffer.indices will be used later
+    let idxVerts = 0;
+    let idxCoord = 0;
+    for (let faceIdx = 0; faceIdx < nbFaces; ++faceIdx) {
+      const nbFaceVerts = buffer.indices[idxVerts++];
+      const texcoods = buffer.faceVertexUvs[idxCoord++];
+      if (nbFaceVerts * 2 === texcoods.length) {
```

Comment:
if it is false, shouldn't you increment idxVerts anyway ?

Reply:
Yes, you are right.